Gemfile

```diff
@@ -29,13 +29,7 @@ group :development do
   gem "puppet-module-win-default-r#{minor_version}", '~> 0.3',   require: false, platforms: [:mswin, :mingw, :x64_mingw]
   gem "puppet-module-win-dev-r#{minor_version}", '~> 0.3',       require: false, platforms: [:mswin, :mingw, :x64_mingw]
   gem "github_changelog_generator",                              require: false, git: 'https://github.com/skywinder/github-changelog-generator', ref: '20ee04ba1234e9e83eb2ffb5056e23d641c7a018' if Gem::Version.new(RUBY_VERSION.dup) >= Gem::Version.new('2.2.2')
-end
-group :system_tests do
-  gem "puppet-module-posix-system-r#{minor_version}", require: false, platforms: [:ruby]
-  gem "puppet-module-win-system-r#{minor_version}",   require: false, platforms: [:mswin, :mingw, :x64_mingw]
-  gem "beaker-testmode_switcher", '~> 0.4',           require: false
-  gem "master_manipulator",                           require: false
-  gem "puppet-blacksmith", '~> 3.4',                  require: false
+  gem 'pdk', git: 'git://github.com/puppetlabs/pdk.git'
```

Comment:
looks un-needed

Reply:
oh yeah, that was debugging from my side and needs to be removed before merge

Reply:
done.

Reply:
this was a workaround for PDK-1525